pyfr/plugins/tavg.py

```diff
@@ -351,3 +357,200 @@ def __call__(self, intg):
                     self.tstart_acc = intg.tcurr
 
                 self.tout_last = intg.tcurr
+
+
+class TavgCLIPlugin(BaseCLIPlugin):
+    name = 'tavg'
+    systems = ['*']
+    formulations = ['dual', 'std']
+
+    @classmethod
+    def parse_cli(cls, parser):
+        cmd = cls.name
+        cp = parser.add_parser(cmd, help=f'{cmd} --help')
+        sp = cp.add_subparsers(dest='sub_cmd')
+
+        # Merge command
+        scmd = 'merge'
+        ssp = sp.add_parser(scmd, help=f'{cmd} {scmd} --help')
+        ssp.set_defaults(process=TavgCLIPlugin.merge_cli)
+        ssp.add_argument('solns', nargs='*', help='Averages to merge')
+        ssp.add_argument('output', help='Output file name')
+
+    @classmethod
+    def merge_cli(cls, args):
```

Comment:
Can probably handle this with a decorator so you don't need both the class and regular method to be explicitly defined.

```
@cli_plugin_method
def merge(self, args):
```